src/program/lwaftr/tests/propbased/genyang.lua

```diff
@@ -3,24 +3,31 @@ module(..., package.seeall)
 -- This module provides functions for generating snabb config
 -- commands with random path queries and values
 
-local ffi    = require("ffi")
-local schema = require("lib.yang.schema")
+local ffi       = require("ffi")
+local schema    = require("lib.yang.schema")
+local data      = require("lib.yang.data")
+local path      = require("lib.yang.path")
```

Comment:
I don't think this is being used. It's also imported in selftest where I don't believe it's used.